src/Ball.java

```diff
@@ -56,18 +58,17 @@ public float getDiameter() {
     }
 
     public float getX() {
-        return x;
+        return x; // this decides where on the graph the ball starts
     }
 
     public float getY() {
         return y;
     }
 
-    // Setters that you need go here - by default, only colors
+    // I called on a predefined method. I input the values of r,g,b in sketch and they are then run through this method.
 
-    public void setColors(int fill, int border) {
-        borderColor = border;
-        fillColor = fill;
+    public void setColors( int r, int g, int b) {
```

Comment:
The only negative of adjusting the mutator in this way is now you can't set the `borderColor` value, and also the name is confusing since you are only setting ONE color, not plural colors. Maybe renaming this `setFillColor` and creating a matching `setBorderColor` would have been clearer design